Dockerfile.konflux

```diff
@@ -0,0 +1,72 @@
+
+# Build arguments
+ARG SOURCE_CODE=.
+
+FROM registry.redhat.io/ubi8/go-toolset@sha256:4ec05fd5b355106cc0d990021a05b71bbfb9231e4f5bdc0c5316515edf6a1c96 AS stage
```

Comment:
can you please explain what this stage is used for?

Reply:
why the base image h ere is go-toolset?

Reply:
Yes, it should be ubi8/ubi-minimal.Updated it